static/src/stylesheets/module/_release-message.scss

```diff
@@ -473,9 +473,9 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 .site-message--membership-message-deep-purple {
     background: #7d0068;
 }
-.site-message--membership-message-orange {
+.site-message--membership-message-yellow {
```

Comment:
The orange used as one of the Membership message colours is actually used by Contributions, so Ben Wuersching asked that we replace the membership colour with yellow and use the orange (#ffbb00) for the Contributions banner.